website/docs/r/lb.html.markdown

```diff
@@ -54,7 +54,7 @@ Terraform will autogenerate a name beginning with `tf-lb`.
 * `access_logs` - (Optional) An Access Logs block. Access Logs documented below.
 * `subnets` - (Optional) A list of subnet IDs to attach to the LB. Subnets
 cannot be updated for Load Balancers of type `network`. Changing this value
-will for load balancers of type `network` will force a recreation of the resource. 
+will for load balancers of type `network` will force a recreation of the resource.
```

Comment:
Do you mind to fix typo here in the mean time? I think there's an extra `will`. It probably should be:

Changing this value for load balancers of type `network` will force a recreation of the resource.